src/main/java/com/epam/lab/library/controller/ItemAjaxController.java

```diff
@@ -0,0 +1,222 @@
+package com.epam.lab.library.controller;
+
+
+import com.epam.lab.library.dao.BookDaoImpl;
+import com.epam.lab.library.dao.ItemDaoImpl;
+import com.epam.lab.library.dao.StatusesDaoImpl;
+import com.epam.lab.library.dao.UserDaoImpl;
+import com.epam.lab.library.dao.interfaces.BookDao;
+import com.epam.lab.library.dao.interfaces.ItemDao;
+import com.epam.lab.library.dao.interfaces.StatusesDao;
+import com.epam.lab.library.dao.interfaces.UserDao;
+import com.epam.lab.library.domain.Item;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.sql.Date;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@WebServlet(loadOnStartup = 1)
+public class ItemAjaxController extends HttpServlet {
+
+    private ItemDao itemDao = new ItemDaoImpl();
+    private BookDao bookDao = new BookDaoImpl();
+    private UserDao userDao = new UserDaoImpl();
+    private StatusesDao statusesDao = new StatusesDaoImpl();
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+
+        String[] pathInfo = req.getPathInfo().split("/");
+        if (pathInfo[1].equals("get")) {
```

Comment:
When you compare something with a constant, you should always put constant first to avoid NPE for sure